docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/03_LORIS_MRI_Install/Signal_Viewer_UI_Install.md

```diff
@@ -0,0 +1,81 @@
+# EEG Visualization Install
+
+> Last update: 2022-11-02
+
+## Context
+
+This document is a guide to install and configure Loris to use the `EEG Visualization` features.
+
+## TOC
+
+1. Install
+    1. [Environment](#environment)
+    1. [Installation requirements](#installation-requirements)
+1. [Import EEG BIDS data to Loris](#import-eeg-bids-data-to-loris)
+
+## Install
+
+### Environment
+
+- `Loris` `v24.0` or above.
+- `Loris-MRI` installed.
+
+
+### Installation requirements
+
+Follow the [installation steps for the visualization features](https://github.com/aces/Loris/tree/main/modules/electrophysiology_browser#-installation-requirements-to-use-the-visualization-features).
+
+
+## Import EEG BIDS data to Loris
+
+Adapt and use the following `pipeline.sh` script. It will:
+- take BIDS archives located in `/data/incoming`
+- extract them to their folder in `/data/processing/bidsFiles/`
+- move the archive to `/data/archive`
+- import data into Loris DB with `/opt/$PROJECT_NAME/bin/mri/python/bids_import.py`
+
+```bash
```

Comment:
shouldn't this script be in LORIS-MRI in the tools directory instead of in the README? And the README would point to the script in LORIS-MRI?

Especially when LORIS-MRI can be installed in a different VM/Container than LORIS.